### ISSUE TITLE
Copy-edit the documentation

### ISSUE THREAD

**Issue by jepusto** [jepusto/metaselection#41]
In the documentation, there is inconsistent use of punctuation (i.e., periods) at the end of phrases.

**Comment by meghapsimatrix** [comment]
do we want the . at end or no? @jepusto @mcitkowicz

**Comment by jepusto** [comment]
Yes, I vote the input parameter descriptions should each end with a period. This seems to be the style of base R documentation (e.g., `?lm`).
